components/net/resource_thread.rs

```diff
@@ -236,6 +236,10 @@ impl ResourceChannelManager {
                     http_state,
                 ),
             },
+            CoreResourceMsg::DeleteCookies() => {
+                http_state.cookie_jar.write().unwrap().clear_storage();
```

Comment:
According to https://w3c.github.io/webdriver/#dfn-delete-cookies (which is invoked from https://w3c.github.io/webdriver/#dfn-delete-all-cookies) this can delete more cookies than we want. We will want a message that includes the list of cookies to delete and the URL associated with the deletion request.

Reply:
> According to https://w3c.github.io/webdriver/#dfn-delete-cookies (which is invoked from https://w3c.github.io/webdriver/#dfn-delete-all-cookies) this can delete more cookies than we want. We will want a message that includes the list of cookies to delete and the URL associated with the deletion request.

I think we dont need to give a list of cookies and the URL associated since the [spec](https://w3c.github.io/webdriver/#dfn-delete-all-cookies) we need to delete all the cookies pertaining to a particular document. (I dont find the need of a URL)

Reply:
You're right about one part; https://w3c.github.io/webdriver/#dfn-associated-cookies means that we do not need to create a list of cookies to delete ahead of time. However,  https://w3c.github.io/webdriver/#dfn-delete-cookies means that we some way of indexing into the cookie storage (see how CookieStorage::cookies_for_url does that).

Reply:
We still need to accept a URL in this message and use that to clear only the cookies for that document.